zerver/tests/test_bots.py

```diff
@@ -1062,7 +1106,8 @@ def test_patch_bot_via_post(self) -> None:
             'full_name': 'Fred',
             'method': 'PATCH'
         }
-        result = self.client_post("/json/bots/hambot-bot@zulip.testserver", bot_info)
+        email = 'hambot-bot@zulip.testserver'
+        result = self.client_patch("/json/bots/{}".format(self.get_bot_user(email).id), bot_info)
```

Comment:
The test coverage issue was this -- the whole point of this test was to use the wrong method, `self.client_post`, here.

Reply:
(In the future, it's best to be careful not to make little cleanups like this in the same commit as other changes, since your not doing so here is what made this little tweak unexpectedly breaking this hard to debug...)